packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -1311,6 +1313,26 @@ abstract class RenderSliver extends RenderObject {
   @override
   void handleEvent(PointerEvent event, SliverHitTestEntry entry) { }
 
+  /// Whether the [SemanticsNode]s associated with this [RenderSliver] should
+  /// be excluded from the semantic scrolling area.
+  ///
+  /// [RenderSliver]s that stay on the screen even though the user has scrolled
+  /// past them (e.g. a pinned app bar) should set this to `true`.
+  set excludeFromSemanticsScrolling(bool value) {
+    if (_excludeFromSemanticsScrolling == value)
+      return;
+    _excludeFromSemanticsScrolling = value;
+    markNeedsSemanticsUpdate();
```

Comment:
onlyChanges can never be true for this because `excludeFromSemanticsScrolling` affects the shape of the semantic tree (whether this thing is sorted into the first or the second plane of the RenderSemanticsGestureHandler) and for those type of changes onlyChanges has to be false.